sync/syncer.go

```diff
@@ -196,7 +196,12 @@ func (s *Syncer) lastTickedLayer() types.LayerID {
 
 func (s *Syncer) Synchronise() {
 	defer s.syncRoutineWg.Done()
+
 	currentSyncLayer := s.lValidator.ValidatedLayer() + 1
+	if currentSyncLayer == 1 && s.lastTickedLayer() == 1 { // skip validation for first layer
+		s.Info("Skipping validation of layer 1")
```

Comment:
why not 
if s.currentLayer == 1 {
		return
	}
?
(currentLayer is what we get from the clock)

Reply:
the print should be more like "dont sync on layer one" or "wait for layer bigger then 1 to sync"
saying that we skip the validation is incorrect it will happen